release/release-trueos.sh

```diff
@@ -161,6 +161,23 @@ clean_poudriere()
 	echo -e "y\n" | poudriere ports -d -p ${POUDRIERE_PORTS}
 }
 
+super_clean_poudriere()
+{
+	#Look for any leftover mountpoints/directories and remove them too
+	for i in `ls ${POUDRIERED_DIR}/*-make.conf`
+	do
+		name=`basename "${i}" | rev | cut -d "-" -f 2-10 | rev`
+		if [ ! -d "${POUDRIERED_DIR}/jails/${name}" ]  && [ -d "${POUDRIERE_BASEFS}/jails/${name}" ] ; then
+			#Jail configs missing, but jail mountpoint still exists
+			#Need to completely destroy the old jail dataset/dir
+			_stale_dataset=$(mount | grep 'on ${POUDRIERE_BASEFS}/jails/${name} ' | cut -w -f 1)
+			umount ${POUDRIERE_BASEFS}/jails/${name}
+			rmdir ${POUDRIERE_BASEFS}/jails/${name}
+			zfs destroy -r ${_stale_dataset}
```

Comment:
Need some error checking here. You probably want to validate _stale_dataset exists or isn't bogus before you start running zfs destroy.

Reply:
done!